tests/test_tifffile.py

```diff
@@ -15332,7 +15354,7 @@ def test_write_bitspersample_fail():
             with pytest.raises(ValueError):
                 tif.write(
                     data.astype(numpy.uint8),
-                    bitspersample=4,
```

Comment:
Is it important that this test case (with bitpersample=4) fails? That would make things a bit more complicated.